openmicroscopy/openmicroscopy
# Big image export jpg config

This is the final commit from https://github.com/openmicroscopy/openmicroscopy/pull/4223 that was removed from that PR to make the PR equivalent of the 5.1.4 PR.

This PR adds the configuration of size for download of image as jpg/png/tiff.

To test:
 - Images bigger than 12k * 12k pixels (144000000) should be disabled for Download as Jpg, Png & Tiff.
 - Configure with a different number of pixels. E.g. 3000 x 3000 = 9000000
```
$ bin/omero config set omero.client.download_as.max_size 9000000
$ bin/omero admin restart
```
Now, images larger than this new size should be disabled for download.
